webpack.config.js

```diff
@@ -20,7 +20,7 @@ const betaEnv = process.env.BETA_ENV;
 const nodeEnv = process.env.NODE_ENV;
 
 // Set `true` for testing cloud-services-config https://github.com/RedHatInsights/cloud-services-config#testing-your-changes-locally
```

Comment:
I think this comment should be removed as we won't set here... but possibly move the link to the prompt message or to a console output at the end of the build if the option is selected?

Reply:
I moved the comment alongside the route code.